### PR TITLE
fix(MusicPlaylist): add trackItemList property

### LINKED ISSUE #126: MusicPlaylist only links to MusicRecording, not ItemList
The [schema definition for MusicPlaylist](https://schema.org/MusicPlaylist) says that the `track` field can be either a MusicRecording or an ItemList.
The [graphql definition](https://github.com/MTG/ce-api/blob/7b88001ec82f627dd75322edcc5012e38ec100c2/src/schema/type/MusicPlaylist.graphql#L249) only appears to link to a MusicRecording. We need to be able to also link to a list here.

### PR DESCRIPTION
**Pull request by ChristiaanScheermeijer**
This PR adds a `trackItemList` property to the `MusicPlaylist` type.

Fixes #126

### CODE REVIEW

File: src/schema/type/MusicPlaylist.graphql [hunk 1]
```diff
@@ -213,4 +213,6 @@ type MusicPlaylist implements SearchableInterface & ThingInterface & ProvenanceE
   numTracks: Int #aggregate from track property
   "https://schema.org/track"
   track: MusicRecording @relation(name: "MUSIC_RECORDING", direction: "OUT")
+  "https://schema.org/track"
+  trackItemList: ItemList @relation(name: "ITEM_LIST", direction: "OUT")
```
Comment (alastair) [inline src/schema/type/MusicPlaylist.graphql:217]: maybe we should make this relation name a bit more explicit? `TRACK_ITEM_LIST`?

Comment (alastair) [reply src/schema/type/MusicPlaylist.graphql:217]: This is one of the cases where we might need special consideration in the json-ld renderer. In this case we should add one or the other, but not both, and the value of the `track` field would have a different format

**Review by alastair — COMMENTED** [review]
I see the two specific Merge mutations, so everything looks good here. One small question about naming below
Screenshot 2021-02-26 at 12 44 58 PM

File: src/schema/type/MusicPlaylist.graphql (hunk 1 above)
Comment (ChristiaanScheermeijer) [reply src/schema/type/MusicPlaylist.graphql:217]: Yes good point, I blindly followed the convention from above

**Review by alastair — APPROVED** [review]

**Pull request merged**